src/shared/utils/exportData/exportDataSucceed.ts

```diff
@@ -27,16 +27,18 @@ const exportProcessedData = async ({
   abTrailsItemsData,
   flankerItemsData,
   suffix,
-}: AppletExportData & { suffix: string }) => {
+  enableDataExportRenaming,
+}: AppletExportData & { suffix: string; enableDataExportRenaming: boolean }) => {
   await exportTemplate({
     data: reportData,
-    fileName: GENERAL_REPORT_NAME + suffix,
-    defaultData: reportData.length > 0 ? null : reportHeader,
+    fileName: getGeneralReportName(enableDataExportRenaming) + suffix,
+    defaultData: reportData.length > 0 ? null : getReportHeader(enableDataExportRenaming),
   });
   await exportTemplate({
     data: activityJourneyData,
     fileName: JOURNEY_REPORT_NAME + suffix,
-    defaultData: activityJourneyData.length > 0 ? null : activityJourneyHeader,
+    defaultData:
+      activityJourneyData.length > 0 ? null : getActivityJourneyHeader(enableDataExportRenaming),
```

Comment:
Same comment as above. Determine the headers here instead of relying on an external function that will be removed when the flag is archived.